sync/course-db.js

```diff
@@ -1273,6 +1273,11 @@ module.exports.loadQuestions = async function (coursePath) {
     validate: validateQuestion,
     recursive: true,
   });
+  for (let qid in questions) {
+    if (qid[0] === '@') {
+      infofile.addError(questions[qid], `Question IDs are not allowed to begin with '@'`);
```

Comment:
For posterity, I think it's important to note that this will still result in a DB row for the given QID. I think this is fine, as a user will never be able to actually *point* to that question from an `infoAssessment.json` file since we'll special-case the handling of QIDs that start with `@` to never refer to questions within the same course.

Reply:
huh, will this cause problems then, if the user then edits the QID but not the UUID?